libtock/tock.h

```diff
@@ -36,7 +36,7 @@ typedef enum {
   TOCK_ERROR_NOSUPPORT   = 9,
   TOCK_ERROR_NODEVICE    = 10,
   TOCK_ERROR_UNINSTALLED = 11,
-  TOCK_ERROR_NOACK       = 12
+  TOCK_ERROR_NOACK       = 12,
```

Comment:
Why not also include `ERROR_BADRVAL` here?

Reply:
BADRVAL is not an error that can be returned by the kernel. It's just used in userspace that the kernel has returned an unexpected return value.